src/librbd/cache/Utils.h

```diff
@@ -4,6 +4,8 @@
 #ifndef CEPH_LIBRBD_CACHE_UTILS_H
 #define CEPH_LIBRBD_CACHE_UTILS_H
 
+#include "acconfig.h"
```

Comment:
This PR will be rebased after https://github.com/ceph/ceph/pull/36624 is merged.